tools/shared/pragma.c

```diff
@@ -1053,7 +1053,29 @@ do_sw(void)
   case SW_ESCTYALIAS:
     break;
   case SW_ALIGN:
-    break;
+    /* declaration align XX */
+    if (gtok() != T_INT) {
+      int backup_nowarn = gbl.nowarn;
+      gbl.nowarn = false;
+      error((error_code_t)280, ERR_Warning, lineno, \
+      "ALIGN: allow int alignment only", 0);
```

Comment:
Suggestion for error message: `ALIGN: non-integer alignment`

Reply:
> Suggestion for error message: `ALIGN: non-integer alignment`

Thank you, I have revised this error message.